src/components/ossGuidance.tsx

```diff
@@ -0,0 +1,77 @@
+import { faGithub } from "@fortawesome/free-brands-svg-icons"
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome"
+import { Link } from "gatsby"
+import React, { CSSProperties } from "react"
```

Comment:
なぜかプッシュできなかったのでちょっとテスト を🙇 
```suggestion
import React from "react"
```